### ISSUE TITLE
Gantt: Excessive path when using simpleConnect algorithm with pathfinder

### ISSUE THREAD

**Issue by danielroach** [highcharts/highcharts#10329]
#### Expected behaviour
Gantt renders the borders around the single row series rather than extra lines.
Also dependency lines shouldn't go up into the xAxis (date) area

#### Actual behaviour
Gantt yAxis renders with lines beyond the expected areas, and the dependency lines go up into the xAxis dates.

![Screen Shot 2019-03-18 at 4 42 26 pm](https://user-images.githubusercontent.com/6583635/54509200-df348e00-499c-11e9-88ef-9f9f976d0490.png)

#### Live demo with steps to reproduce
https://jsfiddle.net/nw8sv7ht/1/

#### Product version

Highcharts-Gantt 7.0.3

#### Affected browser(s)
Google Chrome Version 72.0.3626.121 (Official Build) (64-bit)

**Comment by sebastianbochan** [comment]
Hi @danielroach,
Thank you for reporting about the problem. Let me know about size of your chart and steps to reproduce, because I was not able to replicate the issue.

![ganttline](https://user-images.githubusercontent.com/2115686/54513411-aa521800-4957-11e9-848b-69ce87aa1b9f.gif)

Chrome: 73.0.3683.75

FYI @jon-a-nygaard

**Comment by danielroach** [comment]
Hi @sebastianbochan thanks for your reply,
Oddly today when I look at my fiddle above the depencency line issue isnt happening anymore either, ill see if i can figure out how i reproduced it.

However, the issue with the left table grid drawing those two lines below the the yAxis label is reproducible in your screen snippet,

and that issue is probably more pressing to my use case than the depencency line issue.


heres how that table renders with > 1 row item:

![Screen Shot 2019-03-19 at 12 10 11 pm](https://user-images.githubusercontent.com/6583635/54573318-fd9f9580-4a3f-11e9-9d61-16a742d112cd.png)



heres how it renders with 1 row item:

![Screen Shot 2019-03-19 at 12 10 23 pm](https://user-images.githubusercontent.com/6583635/54573339-0e500b80-4a40-11e9-982e-76ddf6e0b96b.png)

**Comment by danielroach** [comment]
Hi @sebastianbochan I figured out how to reproduce the dependencies line bug,

It seems to occur when you drag the cells to change the date, when you have only 1 row in the gantt.

 
![screen](https://user-images.githubusercontent.com/6583635/54573654-4a37a080-4a41-11e9-98e5-6f569b76be6c.gif)

**Comment by sebastianbochan** [comment]
Hi @danielroach,
Thank you for the feedback.

The first issue (incorrect griddlines) is known bug, reported and fixed here: https://github.com/highcharts/highcharts/issues/10161

Regarding to the second issue it is a bug. 

FYI @jon-a-nygaard

**Comment by jon-a-nygaard** [comment]
Hi, thanks again for reporting the issue.

I have located the bug to be in the `simpleConnect` algorithm of the pathfinder module.
Here's a simplified example of the issue https://jsfiddle.net/jon_a_nygaard/sa9Lkbo3

As a workaround you can consider configuring the chart to use a different algorithm for pathfinder, as in the following example https://jsfiddle.net/jon_a_nygaard/tcm4whav/

**Comment by danielroach** [comment]
Hi @jon-a-nygaard thanks for your workaround.

Could you please provide an indication on when the next release of highcharts might be released that includes the solution to the two issues I described in the original question?

**Comment by sebastianbochan** [comment]
Hi @danielroach,
@jon-a-nygaard is on vacation until tomorrow, but he will answer you by the end of this week.

**Comment by jon-a-nygaard** [comment]
Hi @danielroach,

Sorry for the late reply, there was some misunderstanding with the length of my vacation time.

Too answer your questions in the first comment:

> Gantt renders the borders around the single row series rather than extra lines.

This is a known issue reported at https://github.com/highcharts/highcharts/issues/10161. Its fix was released with the version v7.1.0.

> Also dependency lines shouldn't go up into the xAxis (date) area

I have added a pull request #10620 that fixes this issue, it is currently in review, and unless something is found I expect it to be included with the next maintenance release v7.1.2 within a few weeks.
